lib/scenic/adapters/postgres.rb

```diff
@@ -88,7 +88,7 @@ def drop_view(name)
       # @return [void]
       def create_materialized_view(name, sql_definition)
         raise_unless_materialized_views_supported
-        execute "CREATE MATERIALIZED VIEW #{name} AS #{sql_definition};"
+        execute "CREATE MATERIALIZED VIEW #{quote_table_name(name)} AS #{sql_definition};"
```

Comment:
My general position is that for strings like this, I'd rather have them be a shade long than wrap string creation or use a heredoc with `squish` or something. If you disagree, I'm okay with changing it.